1-js/01-getting-started/2-manuals-specifications/article.md

```diff
@@ -1,42 +1,41 @@
 
-A new specification version is released every year. In-between these releases, the latest specification draft is at <https://tc39.es/ecma262/>.
+ஒவ்வொரு ஆண்டும் ஒரு புதிய திட்ட விவரப் பதிப்புரு (version) வெளியிடப்படுகிறது. இந்த வெளியீடுகளுக்கு இடையில், அண்மை கால திட்ட விவர வரைவை காண <https://tc39.es/ecma262/> இங்கே செல்லவும்.
 
-To read about new bleeding-edge features, including those that are "almost standard" (so-called "stage 3"), see proposals at <https://github.com/tc39/proposals>.
+கிட்டத்தட்ட தர அளவுப்படுத்தப்பட்ட ("நிலை 3" என்று அழைக்கப்படுகிறது) மற்றும் புதிய முன்னணி பண்புகளை (bleeding-edge features) பற்றி படிக்க, இங்கே உள்ள புத்தாய்வுத்திட்டங்களை <https://github.com/tc39/proposals> பார்க்கவும். 
 
-Also, if you're in developing for the browser, then there are other specs covered in the [second part](info:browser-environment) of the tutorial.
+மேலும், நீங்கள் உலாவி (browser) உருவாக்கும் பணியில் இருப்பவராக இருந்தால் , உங்களுக்காக பிற திட்ட விவரங்கள் இந்த தனிமுறை கல்வி குறிப்பின் (tutorial) [இரண்டாம் பக்கத்தில் (second part)](info:browser-environment)  கொடுக்கப்பட்டுள்ளன.
 
-## Manuals
+## கையேடுகள்
 
-- **MDN (Mozilla) JavaScript Reference** is a manual with examples and other information. It's great to get in-depth information about individual language functions, methods etc.
+- **எம்.டி.என்(MDN) (மொஸில்லா-Mozilla) ஜாவாஸ்கிரிப்ட் குறிப்பு** எடுத்துக்காட்டுகளுடன் கூடிய மற்றும் பிற தகவல்களை கொண்டுள்ள ஒரு கையேடு ஆகும். தனிப்பட்ட நிரலாக்க மொழி செயல்பாடுகள், முறைகள் (methods) மற்றும் பிற ஆழமான தகவல்களைத்  தெரிந்து கொள்வது மிகவும் நல்லது.
 
-    One can find it at <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference>.
+    அவைகளை இங்கே காணலாம் <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference>.
 
-    Although, it's often best to use an internet search instead. Just use "MDN [term]" in the query, e.g. <https://google.com/search?q=MDN+parseInt> to search for `parseInt` function.
+    இருப்பினும், அதற்கு பதிலாக இணைய தேடலைப் பயன்படுத்துவது சிறந்ததாக கருதப்படுகிறது. வினவலில் "MDN [term]" ஐப் பயன்படுத்தவும், எடுத்துக்காட்டாக `parseInt` என்ற செயல்பாட்டைத் தேட <https://google.com/search?q=MDN+parseInt>  இவ்வாறு குறிப்பிடவும்.
 
+- **எம்.எஸ்.டி.என் (MSDN)** - என்ற மைக்ரோசாஃப்ட் கையேடு, ஜாவாஸ்கிரிப்ட் (பெரும்பாலும் JScript என குறிப்பிடப்படுகிறது) உட்பட பல தகவல்களைக் கொண்டுள்ளது. இன்டர்நெட் எக்ஸ்ப்ளோரரை பற்றி குறிப்பிட்ட ஏதாவது ஒன்றை தெரிந்துகொள்ள, இங்கே செல்வது நல்லது: <http://msdn.microsoft.com/>.
 
-- **MSDN** – Microsoft manual with a lot of information, including JavaScript (often referred to as JScript). If one needs something specific to Internet Explorer, better go there: <http://msdn.microsoft.com/>.
+    மேலும், "RegExp MSDN" அல்லது "RegExp MSDN jscript" போன்ற சொற்றொடர்களுடன் இணைய தேடலைப் பயன்படுத்தலாம்.
 
-    Also, we can use an internet search with phrases such as "RegExp MSDN" or "RegExp MSDN jscript".
+## ஏற்புடைய அட்டவணைகள்
```

Comment:
Since the **Compatibility tables** are crucial in web development, it's better to mention the English equivalent. 

```suggestion
## ஏற்புடைய அட்டவணைகள் (Compatibility tables)
```
Also, I would love to know the reason behind the translation of compatibility as `ஏற்புடைய` as it's unique 🙂

Reply:
The exact meaning of compatibility is - one should agree with another or it should happen together.

out of ஒத்தியல்பு, உடனியைவு, ஏற்புடைய... I feel "ஏற்புடைய" should be the exact match here.